Lesson4/src/com/home/model/MilitaryUnit.java

```diff
@@ -50,4 +51,13 @@ public String getName() {
         ALREADY_IN,
         SUCCESS;
     }
+
+    public void sortListPerson() {
+        String[] tmpArrPerson = new String[this.personSet.size()];
```

Comment:
you can call personSet instead of this. personSet, it will reference the same object

Reply:
need to just display the list sorted names and surnames or also sort the list Person? Arrays.sort don't work with Objects))

Reply:
no, i mean you can do 
String[] tmpArrPerson = new String[personSet.size()];
instead of 
String[] tmpArrPerson = new String[this.personSet.size()];

Reply:
ok, got it